src/store/filestore.rs

```diff
@@ -0,0 +1,209 @@
+use crate::{
+    error::{FileExists, FileHasNoParent, NoAnnotationFound, NoRegexMatch},
+    model::{from_yaml, to_yaml, Pod},
+    store::Store,
+    util::get_struct_name,
+};
+use colored::Colorize;
+use glob::{GlobError, Paths};
+use regex::Regex;
+use std::{collections::BTreeMap, error::Error, fs, iter::Map, path::PathBuf};
+
+#[derive(Debug)]
+pub struct LocalFileStore {
+    pub directory: PathBuf,
+}
+
+impl Store for LocalFileStore {
+    fn save_pod(&self, pod: &Pod) -> Result<(), Box<dyn Error>> {
+        let class = get_struct_name::<Pod>()?;
+
+        LocalFileStore::save_file(
+            &self.make_annotation_path(
+                &class,
+                &pod.hash,
+                &pod.annotation.name,
+                &pod.annotation.version,
+            ),
+            &serde_yaml::to_string(&pod.annotation)?,
+            true,
+        )?;
+
+        LocalFileStore::save_file(
+            &self.make_spec_path(&class, &pod.hash),
+            &to_yaml::<Pod>(&pod)?,
+            false,
```

Comment:
Would you mind indicating what's being controlled by true/false? I know it's part of the API for save but otherwise unclear and a quick comment would help a lot